src/App.js

```diff
@@ -12,12 +12,8 @@ function App() {
         <Navbar />
       </div>
 
-      {/* below are Progress Bar and Direction Buttons placeholders*/}
-      <div className="progress-bar">progress bar</div>
-      <div className="direction-buttons">direction buttons</div>
-
```

Comment:
May clash with the changes in #5. Where are the direction buttons going to end up?

Reply:
Whoops, I replaced the direction buttons with the forward button and must have forgotten to delete that code fragment